.editorconfig

```diff
@@ -1,11 +1,17 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
Now this pretty much looks like the typical editorconfig examples, except for the `md`/`snap` override